srcpkgs/volctl/template

```diff
@@ -1,6 +1,6 @@
 # Template file for 'volctl'
 pkgname=volctl
-version=0.7.8
+version=0.8.0
 revision=1
 archs=noarch
```

Comment:
Please remove `archs=noarch` as part of the transition away from noarch.

Reply:
Done